src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupContainer.md

```diff
@@ -20,32 +21,41 @@ Get-AzRecoveryServicesBackupContainer [-ContainerType] <ContainerType> [[-Backup
 ```
 
 ## DESCRIPTION
+
 The **Get-AzRecoveryServicesBackupContainer** cmdlet gets a backup container.
 A Backup container encapsulates data sources that are modelled as backup items.
-Set the vault context by using the Set-AzRecoveryServicesVaultContext cmdlet before you use the current cmdlet.
+Set the vault context by using the **Set-AzRecoveryServicesVaultContext** cmdlet before you use the current cmdlet.
```

Comment:
SHouldn't we just tell them to use the VaultId parameter to specify the vault in the cmdlet?  Seems like we shouldn't mention the Set-ValultContext cmdlet if it is being deprecated.

Reply:
I agree. I've changed note regarding vault context and mentioned VaultId parameter there.